composer.json

```diff
@@ -41,7 +41,8 @@
         "league/omnipay": "3.0.1",
         "academe/omnipay-authorizenetapi": "3.0.4",
         "omnipay/stripe": "3.0.1",
-        "waryway/php-traits-library": "^1.0"
+        "waryway/php-traits-library": "^1.0",
+        "paragonie/multi-factor": "0.2.2"
```

Comment:
TODO for me:
import this package to composer.lock without updating other packages